arbitrator/arbutil/src/evm/api.rs

```diff
@@ -150,17 +150,29 @@ macro_rules! derive_math {
 }
 
 #[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Debug, Default)]
+
 #[must_use]
+
 pub struct Gas(pub u64);
 
 derive_math!(Gas);
 
 #[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Debug, Default)]
+
 #[must_use]
+
 pub struct Ink(pub u64);
 
 derive_math!(Ink);
 
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Debug, Default)]
+
+#[must_use]
+
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Debug, Default)]
+
+#[must_use]
```

Comment:
This code is different from official branch: https://github.com/OffchainLabs/nitro/blob/v3.6.7/arbitrator/arbutil/src/evm/api.rs#L152-L162